scripts/setup_for_crt_builder.sh

```diff
@@ -6,6 +6,10 @@ set -x
 # with the CRT builder script (see https://github.com/awslabs/aws-crt-builder).
 # For use only as a pre-build step when using the CRT builder script on CI.
 
+# Enables the use of local dependencies, which are required when building with
+# the CRT builder script.
+export AWS_SWIFT_SDK_USE_LOCAL_DEPS=1
```

Comment:
Releases use the CRT builder as well.  If this is set for a release, then a release will use whatever reference CRT builder retrieves & installs by default, not the pinned versions in `packageDependencies.plist`

Reply:
not entirely,

what this switch do, is basically manipulate 
```
let useLocalDeps = ProcessInfo.processInfo.environment["AWS_SWIFT_SDK_USE_LOCAL_DEPS"] != nil
let useMainDeps = ProcessInfo.processInfo.environment["AWS_SWIFT_SDK_USE_MAIN_DEPS"] != nil

switch (useLocalDeps, useMainDeps) {
case (true, true):
    fatalError("Unable to determine which dependencies to use. Please only specify one of AWS_SWIFT_SDK_USE_LOCAL_DEPS or AWS_SWIFT_SDK_USE_MAIN_DEPS.")
case (true, false):
    package.dependencies += [
        .package(path: "../smithy-swift") <------- THIS
    ]
case (false, true):
    package.dependencies += [
        .package(url: "https://github.com/awslabs/smithy-swift", branch: "main")
    ]
case (false, false):
    package.dependencies += [
        .package(url: "https://github.com/awslabs/smithy-swift", .exact("0.10.0"))
    ]
}
```

change the path to the dir where repo is located in the runner.

This code goes out in released Package.swift and when both AWS_SWIFT_SDK_USE_LOCAL_DEPS and AWS_SWIFT_SDK_USE_MAIN_DEPS are unset, which we expect to be in customer apps, we will use the pinned version.